<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpriteSheetEditorWindow</class>
 <widget class="QMainWindow" name="SpriteSheetEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="GLSpriteWidget" name="openGLWidget">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionImport_SH"/>
    <addaction name="actionExport_SH"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionDump"/>
    <addaction name="separator"/>
    <addaction name="actionExport_sprites_as_PNG"/>
    <addaction name="actionExport_sprites_as_TGA"/>
    <addaction name="actionExport_sprites_IDs"/>
    <addaction name="separator"/>
    <addaction name="actionRename_animation"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_Pick_color"/>
    <addaction name="actionReset_camera"/>
    <addaction name="separator"/>
    <addaction name="actionDisplay_external_references"/>
    <addaction name="actionDisplay_shadow"/>
    <addaction name="actionDisplay_ground"/>
    <addaction name="separator"/>
    <addaction name="actionTimeline"/>
    <addaction name="actionSprite_sheet_view"/>
    <addaction name="actionColor_table_view"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockTimeline">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>296</width>
     <height>150</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>250</height>
    </size>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="windowTitle">
    <string>Timeline</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="15,0,0,0,1,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>23</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_newanimation">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>New animation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPrevious">
         <property name="maximumSize">
          <size>
           <width>23</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPausePlay">
         <property name="maximumSize">
          <size>
           <width>23</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="shortcut">
          <string>Space</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="labelCurentFrame">
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>23</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="maximumSize">
              <size>
               <width>7</width>
               <height>23</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>/</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelTotalFrames">
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>23</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnNext">
         <property name="maximumSize">
          <size>
           <width>23</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QQuickWidget" name="timelineQML">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="resizeMode">
        <enum>QQuickWidget::SizeRootObjectToView</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="spritesView">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>100</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Sprite sheet view</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QQuickWidget" name="quickSpriteView">
       <property name="resizeMode">
        <enum>QQuickWidget::SizeRootObjectToView</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetColorTable">
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Color table view</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QQuickWidget" name="quickColorTable">
       <property name="resizeMode">
        <enum>QQuickWidget::SizeRootObjectToView</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionDump">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Dump</string>
   </property>
  </action>
  <action name="actionExport_sprites_as_PNG">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export sheets as PNG</string>
   </property>
  </action>
  <action name="actionExport_sprites_as_TGA">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export sheets as TGA</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionDisplay_external_references">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display external references</string>
   </property>
  </action>
  <action name="actionDisplay_shadow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display shadow</string>
   </property>
  </action>
  <action name="actionDisplay_ground">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display ground</string>
   </property>
  </action>
  <action name="actionPick_background_color">
   <property name="text">
    <string>Pick background color</string>
   </property>
  </action>
  <action name="action_Pick_color">
   <property name="text">
    <string>Pick background color</string>
   </property>
  </action>
  <action name="actionTimeline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Timeline</string>
   </property>
  </action>
  <action name="actionImport_SH">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Import .SH</string>
   </property>
  </action>
  <action name="actionExport_SH">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export .SH</string>
   </property>
  </action>
  <action name="actionExport_sprites_IDs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export sheets as IDs</string>
   </property>
  </action>
  <action name="actionExport_color_table">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export color table</string>
   </property>
  </action>
  <action name="actionSprite_sheet_view">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sprite sheet view</string>
   </property>
  </action>
  <action name="actionImport_color_table">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import color table</string>
   </property>
  </action>
  <action name="actionColor_table_view">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Color table view</string>
   </property>
  </action>
  <action name="actionReset_camera">
   <property name="text">
    <string>Reset camera</string>
   </property>
  </action>
  <action name="actionRename_animation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rename animation</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>GLSpriteWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>spriteSheetEditor/GLSpriteWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
